src/controllers/loginController.js

```diff
@@ -0,0 +1,29 @@
+const loginService = require('../services/registration');
+
+const loginController = (req, res) => {
+  if (req.body.username && req.body.password) {
+    loginService.authorizeUser(JSON.stringify(req.body))
```

Comment:
is there a reason for stringifying and the parsing to json for this information?
also, its sending the req.body as a parameter. you should create a new object that takes the req.body information, the services shouldn't rely on anything else, just the required data for it's functionality.